README.md

```diff
@@ -1,5 +1,9 @@
 # Joint non-rigid registration and reconstruction of image series
 
+Joint non-rigid registration aims to calculate non-linear deformation artifacts from a series of images taken of an object. 
```

Comment:
Minor nitpicking: "Joint" here refers to registration and reconstruction. So for "joint" to make sense, you'd need to add "and reconstruction".  Since the line before already says that, we can just write "This methods aims..." instead.